### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by eahefnawy**
our sdk is now available on rubygems.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+gem install yellow-sdk
+```
+
+Examples
+---------
+```
+require 'yellow-sdk'
+
+# pass your API key and secret respectively
+yellow = Yellow::Client.new("Eo2XE5SrHcJVdqK9uIDU", "73jnZflRHpGReqFxzwF_JtynRNmZXoQspPaxTtsy")
```
Comment (JamesPiechota) [inline README.md:15]: These aren't real key/secrets right? Or if they are, only on your dev machine?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    "address": "155xsayoDXxRFP9rxDoecmpVUo7y5xKtc7", # Invoice Address
+    "base_ccy": "USD",
+    "base_price": "0.05",
+    "callback": "https://example.com",
+    "expiration": "2015-03-10T18:17:51.248Z", # Each invoice expires after 10 minutes of creation
+    "id": "6dd264975861fddbfe4404ed995f1ca4", # Invoice ID (to query the invoice later if you need to!)
+    "invoice_ccy": "BTC",
+    "invoice_price": "0.00017070",
+    "received": "0",
+    "redirect": null,
```
Comment (JamesPiechota) [inline README.md]: can you remove the 'redirect' field?

File: lib/yellow-sdk.rb [hunk 3]
```diff
@@ -3,4 +3,4 @@
 require 'uri'
 require 'net/http'
 require "openssl"
-require "always_verify_ssl_certificates"
\ No newline at end of file
+#require "always_verify_ssl_certificates"
```
Comment (JamesPiechota) [inline lib/yellow-sdk.rb]: This should be uncommented, right?

File: yellow-sdk.gemspec [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name        = 'yellow-sdk'
   s.summary     = 'Yellow SDK'
-  s.version     = '0.0.1'
+  s.version     = '0.0.4'
```
Comment (JamesPiechota) [inline yellow-sdk.gemspec]: The version above is 0.0.3 - do they have to be in sync?

**Comment by JamesPiechota** [comment]
Looks good, a few minor comments.

File: README.md (hunk 1 above)
Comment (eahefnawy) [inline README.md:15]: yep..they're fakes...copied from our docs

File: README.md (hunk 2 above)
Comment (eahefnawy) [inline README.md]: sure

File: lib/yellow-sdk.rb (hunk 3 above)
Comment (eahefnawy) [inline lib/yellow-sdk.rb]: ooh yeah, it was just giving an error

File: yellow-sdk.gemspec (hunk 4 above)
Comment (eahefnawy) [inline yellow-sdk.gemspec]: good catch ;)

**Pull request merged**